sdk/deviceprovisioningservices/azure-iot-deviceprovisioningservice/azure/iot/deviceprovisioningservice/client.py

```diff
@@ -0,0 +1,155 @@
+# coding=utf-8
```

Comment:
This client module should be private (_client.py) where the ProvisioningServiceClient is only exposed via deviceprovisioningservice \_\_init\_\_.\_\_all\_\_ attribute, similar should apply to aio version (in its corresponding \_\_init\_\_), further simplifying the public surface area.